uint32 enabled
uint32 error_code
string name
string info

